Assets/Scripts/MessageHandler.cs

```diff
@@ -50,6 +50,11 @@ private static void HandleCameraStreamOpenRequest(Rover rover, JObject cameraStr
         camera.StreamWidth = (int)cameraStreamOpenRequest["width"];
         camera.StreamHeight = (int)cameraStreamOpenRequest["height"];
         camera.IsStreaming = true;
+        if (cameraStreamOpenRequest.ContainsKey("intrinsicParameters"))
```

Comment:
As per protocol in the README, the request should always contain `intrinsicParameters`, though the value may be null. Instead of checking if it contains the key, you should check if the value is not null.